app/src/androidTest/java/org/mozilla/focus/activity/MozillaSupportPagesTest.kt

```diff
@@ -127,6 +129,32 @@ class MozillaSupportPagesTest {
         }
         homeScreen {
             verifyTipsCarouselIsDisplayed(false)
+        }.openMainMenu {
+        }.openSettings {
+        }.openMozillaSettingsMenu {
+            switchHomeScreenTips()
+            exitToTop()
+        }
+        homeScreen {
+            closeSoftKeyboard()
+            verifyTipsCarouselIsDisplayed(true)
+            verifyTipText(TestHelper.getStringResource(R.string.tip_fresh_look))
```

Comment:
I see this a bit difficult to read, not sure if there is a simplier way to do the `verifyTipText(tip_fresh_look)` defining that value somewhere outside the test or at the beginning of it...